notebooks/official/training/tpuv5e_gemma_peft_finetuning_and_serving.ipynb

```diff
@@ -154,179 +166,119 @@
     },
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "WReHDGG5g0XY"
+        "id": "7176ea64999b"
       },
       "source": [
-        "#### Set your project ID\n",
+        "### Authenticate your notebook environment (Colab only)\n",
         "\n",
-        "**If you don't know your project ID**, try the following:\n",
-        "* Run `gcloud config list`.\n",
-        "* Run `gcloud projects list`.\n",
-        "* See the support page: [Locate the project ID](https://support.google.com/googleapi/answer/7014113)"
+        "Authenticate your environment on Google Colab.\n"
       ]
     },
     {
       "cell_type": "code",
       "execution_count": null,
       "metadata": {
-        "id": "oM1iC_MfAts1"
+        "id": "7de6ef0fac42"
       },
       "outputs": [],
       "source": [
-        "PROJECT_ID = \"[your-project-id]\"  # @param {type:\"string\"}\n",
+        "import sys\n",
         "\n",
-        "# Set the project id\n",
-        "! gcloud config set project {PROJECT_ID}"
-      ]
-    },
-    {
-      "cell_type": "markdown",
-      "metadata": {
-        "id": "region"
-      },
-      "source": [
-        "#### Region\n",
+        "if \"google.colab\" in sys.modules:\n",
         "\n",
-        "You can also change the `REGION` variable used by Vertex AI. Learn more about [Vertex AI regions](https://cloud.google.com/vertex-ai/docs/general/locations).\n",
+        "    from google.colab import auth\n",
         "\n",
-        "TPUv5e is available in the [following regions listed here](https://cloud.google.com/tpu/pricing)"
-      ]
-    },
-    {
-      "cell_type": "code",
-      "execution_count": null,
-      "metadata": {
-        "id": "region"
-      },
-      "outputs": [],
-      "source": [
-        "REGION = \"us-west1\"  # @param {type: \"string\"}"
+        "    auth.authenticate_user()"
       ]
     },
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "sBCra4QMA2wR"
+        "id": "909aa61d1f13"
       },
       "source": [
-        "### Authenticate your Google Cloud account\n",
+        "### Kaggle credentials\n",
+        "Gemma models are hosted by Kaggle. To use Gemma, request access on Kaggle:\n",
+        "\n",
+        "* Sign in or register at [kaggle.com](https://www.kaggle.com)\n",
+        "* Open the [Gemma model card](https://www.kaggle.com/models/google/gemma) and select \"Request Access\"\n",
+        "* Complete the consent form and accept the terms and conditions\n",
+        "\n",
+        "Then, to use the Kaggle API, create an API token:\n",
         "\n",
-        "Depending on your Jupyter environment, you may have to manually authenticate. Follow the relevant instructions below."
+        "* Open [Kaggle settings](https://www.kaggle.com/settings)\n",
+        "* Select \"Create New Token\"\n",
+        "* A kaggle.json file is downloaded. It contains your Kaggle credentials. Note the username and key as you need to populate this later"
```

Comment:
suggested edit:

- "Take note of the username and key since you'll use these later."

Reply:
Made the change, please review